apph.adoc

```diff
@@ -464,17 +453,16 @@ For each profile, there is a single time, lat and lon.
 A data variable may contain a collection of profile features.
 The instance dimension in the case of profiles specifies the number of profiles in the collection and is also referred to as the **profile dimension**.
 The instance variables, which have just this dimension, including latitude and longitude for example, are also referred to as **profile variables** and are considered to be information about the profiles.
-It is strongly recommended that there always be a profile variable (of any data type) with **`cf_role`** attribute " **`profile_id`** ", whose values uniquely identify the profiles.
+It is strongly recommended that there always be a profile variable (of any data type) with `cf_role` attribute " `profile_id` ", whose values uniquely identify the profiles.
 The profile variables may contain missing values.
 This allows one to reserve space for additional profiles that may be added at a later time, as discussed in section 9.6.
 All the representations described in section 9.1.3 can be used for profiles.
-The global attribute **`featureType={rdquo}profile{rdquo}`** (case-insensitive) should be included if all data variables in the file contain profiles.
-
+The global attribute `featureType={rdquo}profile{rdquo}` (case-insensitive) should be included if all data variables in the file contain profiles.
 
 ==== Orthogonal multidimensional array representation of profiles
 
 If the profile instances have the same number of elements and the vertical coordinate values are identical for all instances, you may use the orthogonal multidimensional array representation.
-This has either a one-dimensional coordinate variable, z(z), provided the vertical coordinate values are in strict monotonic order, or a one-dimensional auxiliary coordinate variable, alt(o), where o is the element dimension.
+This has either a one-dimensional coordinate variable, `z(z)`, provided the vertical coordinate values are ordered monotonically, or a one-dimensional auxiliary coordinate variable, `alt(o)`, where `o` is the element dimension.
```

Comment:
Is changing "in strict monotonic order" to "ordered monotonic" really intended here (then why?) or is it just old text lingering here?

Reply:
Good catch @larsbarring - thank you, it's been updated!